spec/cucumber/rake/task_spec.rb

```diff
@@ -7,6 +7,15 @@
 module Cucumber
   module Rake
     describe Task do
+      describe '@task_name' do
```

Comment:
As this would be an instance method, tradition dictates this would probably be `#task_name` (Possibly just a typo on your behalf)